Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -17,30 +21,115 @@ public class SettingsMenu : PlatformDependentComponent
-            // adds the ExitGame method to the button
-            exitButton.onClick.AddListener(ExitGame);
-            // sets the text of the scrollview
-            settingsMenuGameObject.transform.Find("KeybindingsPanel/KeybindingsText/Viewport/Content").gameObject.MustGetComponent<TextMeshProUGUI>().text = KeyBindings.GetBindingsText();
+            // adds the ExitGame method to the exit button
+            settingsMenuGameObject.transform.Find("ExitPanel/Buttons/Content/Exit").gameObject.MustGetComponent<Button>()
+                .onClick.AddListener(ExitGame);
+
+            shortNameOfBindingToLabel = new Dictionary<string, TextMeshProUGUI>();
+
+            // Displays all bindings grouped by their category.
+            foreach (IGrouping<KeyBindings.KeyActionCategory, KeyValuePair<KeyBindings.KeyAction, KeyBindings.KeyActionDescriptor>> group
+                in KeyBindings.AllBindings().GroupBy(binding => binding.Value.Category))
+            {
+                // Creates a list of keybinding descriptions for the given category.
+                GameObject scrollView = PrefabInstantiator.InstantiatePrefab(ScrollPrefab, Canvas.transform, false).transform.gameObject;
+                scrollView.transform.SetParent(settingsMenuGameObject.transform.Find("KeybindingsPanel/KeybindingsText/Viewport/Content"));
+                // set the titles of the scrollViews to the scopes
+                TextMeshProUGUI groupTitle = scrollView.transform.Find("Group").gameObject.MustGetComponent<TextMeshProUGUI>();
+                groupTitle.text = $"{group.Key}";
+
+                foreach (var binding in group)
+                {
+                    GameObject keyBindingContent = PrefabInstantiator.InstantiatePrefab(KeyBindingContent, Canvas.transform, false).transform.gameObject;
+                    keyBindingContent.transform.SetParent(scrollView.transform.Find("Scroll View/Viewport/Content"));
+
+                    // set the text to the short name of the binding
+                    TextMeshProUGUI bindingText = keyBindingContent.transform.Find("Binding").gameObject.MustGetComponent<TextMeshProUGUI>();
+                    // The short name of the binding.
+                    bindingText.text = binding.Value.Name;
+                    // set the label of the key button
+                    TextMeshProUGUI key = keyBindingContent.transform.Find("Key/Text (TMP)").gameObject.MustGetComponent<TextMeshProUGUI>();
+                    // The name of the key code bound.
+                    key.text = binding.Value.KeyCode.ToString();
+                    shortNameOfBindingToLabel[binding.Value.Name] = key;
+                    // add the actionlistener to be able to change the key code of a binding.
+                    keyBindingContent.transform.Find("Key").gameObject.MustGetComponent<Button>().onClick.AddListener(() => { StartRebindFor(binding.Value); });
```

Comment:
I think the brackets are unnecessary:
```suggestion
                    keyBindingContent.transform.Find("Key").gameObject.MustGetComponent<Button>().onClick.AddListener(() => StartRebindFor(binding.Value));
```